mvc-framework/core/middlewares/cors/corsMiddleware.ts

```diff
@@ -30,16 +29,19 @@ interface MiddlewareData {
     useDefaultCors: boolean
 }
 
-export const handleCors = (requestContext: Mandarine.Types.RequestContext, data: MiddlewareData) => {
+export const handleCors = (requestContext: Mandarine.Types.RequestContext, data: MiddlewareData): boolean => {
     let { corsOptions, useDefaultCors } = data;
-    if(!corsOptions && useDefaultCors) corsOptions = Mandarine.Defaults.MandarineDefaultCorsOptions;
```

Comment:
We should definitely use default configuration for CORS. It's going to be up to the user to override that configuration, but Mandarine should provide something out of the box which should be handled by default. Just like Express JS does or Spring Boot in Java 
![image](https://user-images.githubusercontent.com/60560109/113028192-ff38e380-9158-11eb-87e0-2b22e7c43a8f.png)